app/index.html

```diff
@@ -13,7 +14,41 @@ <h1>HELLO WORLD!</h1><i class="fa fa-hand-spock-o" aria-hidden="true"></i>
   	<div class="header header__second"></div>
   	<div class="header header__third"></div>
   	<div class="header header__fourth"></div>
-  	<div class="header header__fifth"></div>
+  	<div class="header header__fifth">
+		<main class="fifth">
+			<nav class="fifth__menu">
+				<img alt="Logo UI" src="img/logo.png" class="fifth__menu--logo">
+				<ul class="fifth__navigation">
+					<li class="fifth__navigation--item"><a href="#" >PRODUCTS</a></li>
+					<li class="fifth__navigation--item"><a href="#">PRICES</a></li>
+					<li class="fifth__navigation--item"><a href="#">ABOUT US</a></li>
+					<li class="fifth__navigation--item"><a href="#">CONTACTS</a></li>
+				</ul>
+			</nav>
+			<h2 class="fifth__title">Carefully Crafted UI Design Assets</h2>
+			<p class="fifth__subsection">
+					The sight of the tumblers restored Bob Sawyer to a degree of 
+					equanimity which he had not possessed since his interview with 
+					his landlady. His face brightened up, and he began to feel quite 
+					convivial.
+			</p>
+			<button class="fifth__btn">Let's start!</button>
+		</main>
+		<div class="fifth__touches">
+			<div class="fifth__touch fifth__touch--active"></div>
+			<div class="fifth__touch"></div>
+			<div class="fifth__touch"></div>
+			<div class="fifth__touch"></div>
+		</div>		
+		<div class="fifth__arrow fifth__arrow--left">
```

Comment:
please, use here `button` tag, because arrows are buttons and user interacts with them by clicking

Reply:
Okey, I agree - corected.